src/i18n/en/index.ts

```diff
@@ -166,6 +166,99 @@ const en = {
+                name: 'A unique name for your WireGuard tunnel to identify it easily.',
+                pubkey:
+                  'The public key associated with the WireGuard tunnel for secure communication.',
+                prvkey:
+                  'The private key associated with the WireGuard tunnel for secure communication.',
+                address:
+                  'The IP address assigned to this WireGuard client within the VPN network.',
+                serverPubkey:
+                  'The public key of the WireGuard server for secure communication.',
+                allowedIps:
+                  'A comma-separated list of IP addresses or CIDR ranges that are allowed for communication through the tunnel.',
+                endpoint:
+                  'The address and port of the WireGuard server, typically in the format "hostname:port".',
+                dns: 'The DNS (Domain Name System) server that the WireGuard tunnel should use for name resolution.',
+                persistentKeepAlive:
+                  'The interval (in seconds) for sending periodic keep-alive messages to ensure the tunnel stays active. Adjust as needed.',
+                routeAllTraffic:
+                  'If enabled, all network traffic will be routed through the WireGuard tunnel.',
+                preUp:
+                  'Shell commands or scripts to be executed before bringing up the WireGuard tunnel.',
+                postUp:
+                  'Shell commands or scripts to be executed after bringing up the WireGuard tunnel.',
+                preDown:
+                  'Shell commands or scripts to be executed before tearing down the WireGuard tunnel.',
+                postDown:
+                  'Shell commands or scripts to be executed after tearing down the WireGuard tunnel.',
+              },
+              submit: 'Add Tunnel',
+              messages: {
+                configError: 'Error parsing config file',
+                addSuccess: 'Tunnel added',
+                addError: 'Creating tunnel failed',
+              },
+              controls: {
+                importConfig: 'Import Config File',
+                generatePrvkey: 'Generate Private Key',
+              },
+            },
+          },
+          guide: {
+            title: 'Adding WireGuard tunnel',
+            subTitle: `<p>To establish secure communication between two or more devices over the internet create a virtual private network by configuring your tunnel.</p><p>If you don’t see options like Table or MTU it means we do not support it for now, but will be added later.</p>`,
+            card: {
+              title: 'Setting Up A new Tunnel:',
+              content: `
+                <p>1. Import Configuration File</p>
+                <div>
+                <ul>
```

Comment:
broken ul tags, you can write MD here as well and don't deal with HTML that way.